pkgs/tools/networking/linkchecker/default.nix

```diff
@@ -1,54 +1,50 @@
-{ stdenv, lib, fetchFromGitHub, fetchpatch, python2, gettext }:
+{ stdenv, lib, fetchFromGitHub, fetchpatch, python2, python2Packages, gettext }:
+/*
 let
-  # pin requests version until next release.
-  # see: https://github.com/linkcheck/linkchecker/issues/76
-  python2Packages = (python2.override {
-    packageOverrides = self: super: {   
-      requests = super.requests.overridePythonAttrs(oldAttrs: rec {
-        version = "2.14.2";
-        src = oldAttrs.src.override {
-          inherit version;
-          sha256 = "0lyi82a0ijs1m7k9w1mqwbmq1qjsac35fazx7xqyh8ws76xanx52";
-        };
-      });
-    };
-  }).pkgs;
+  argparse = python2Packages.ConfigArgParse;
 in
+*/
 python2Packages.buildPythonApplication rec {
   pname = "LinkChecker";
-  version = "9.3.1";
+  version = "9.4.0";
 
   nativeBuildInputs = [ gettext ];
   pythonPath = (with python2Packages; [
+    ConfigArgParse
+    argcomplete
+    dnspython
+    pyxdg
     requests
   ]) ++ [ gettext ];
 
-  checkInputs = with python2Packages; [ pytest ];
+  checkInputs = with python2Packages; [
+    parameterized
+    pytest
+  ];
 
   # the original repository is abandoned, development is now happening here:
```

Comment:
This can also go. This source is now the proper upstream.

And conventionally, ```src``` goes after ```pname``` and ```version```.